test/views/declaration/OfficeOfExitStandardViewSpec.scala

```diff
@@ -54,114 +54,106 @@ class OfficeOfExitStandardViewSpec extends ViewSpec with OfficeOfExitMessages wi
 
       "display page title" in {
 
-        getElementByCss(createView(), "title").text() must be(messages(title))
+        getElementByCss(createView(), "title").text() mustBe messages(title)
       }
 
       "display section header" in {
 
-        getElementById(createView(), "section-header").text() must be("Locations")
+        getElementById(createView(), "section-header").text() mustBe "Locations"
```

Comment:
Just my opinion but I think we should avoid doing changes like this as it costs time without really any added value. No need to revert as what's done is done

Reply:
I guess this is one to discuss - removing brackets makes tests much easier to read imho.